pkg/service/kubernetes_auth.go

```diff
@@ -25,18 +26,22 @@ var (
 	}
 )
 
-func init() {
-	synv1alpha1.SchemeBuilder.AddToScheme(scheme.Scheme)
+func createCache() *lruCache.Cache {
+	cache, err := lruCache.NewWithEvict(128, nil)
```

Comment:
Would it make sense to make the size configurable via env var?

Reply:
added. WDYT now?